pkg/api/networkservice/mechanisms/kernel/helpers.go

```diff
@@ -57,3 +62,11 @@ func (m *mechanism) GetNetNSInode() string {
 	}
 	return m.GetParameters()[common.NetNSInodeKey]
 }
+
+// GetPCIAddress returns PCI address of the VF device
```

Comment:
Can we not refer to VF here, we might very well talk of a non-SR-IOV enabled device and still use this mechanism.

Reply:
Very true in this case, definitely will fix.